edge/discussion_materials/onnx-edge-scope and profiling.md

```diff
@@ -0,0 +1,75 @@
+
+In another scenario which requires low latency like auto pilot cars. A notable applications include online gaming. Game servers are running in the cloud and the rendered video is transferred to lightweight clients such as AR devices (mobile phone), VR glasses, etc. Conventional cloud games may suffer from high latency and insufficient bandwidth. As real-time games have strict constraints on latency, processing game simulation at the edge node is necessary for the immersive game plays.
+
+
+## Edge device
+
+An edge device is a device which provides an entry point into enterprise or service provider core networks. Examples include routers, routing switches, terminals and other devices. Edge devices also provide connections into carrier and internet service provider networks.
+
+In general, edge devices are normally routers and terminals that provide authenticated access to faster, more efficient backbone and core networks. Edge device is usually smart and has access to the core devices, often include Quality of Service (QoS) and multi-service functions to manage different types of traffic.
+
+## ONNX Edge profile
+
+The work of the ONNX Edge profile includes:
+
+1. Attributes / characteristics: Power, Size, accuracy, speed, Security, etc.
+
+2. ONNX operations subset
+
+3. Other ONNX related limitations 
+
+Work to define strict subsets of the operator sets which apply to edge devices which avoid complex operations imposed by limitations on edge devices with high computational complexity. Exclude use-cases not applicable to the edge (e.g. large scale training scenarios)
+
+
+###Metrics for ONNX Edge
+
+#### Accuracy
+
+The ONNX model zoo collected many wide-used models for image classification, face recognition and image segmentation. While the accuracy number varies, the top-5 accuracy seems acceptable with VGG, resent and mobile net are all above or close to 90%. Accuracy is critical for scenarios like smart city, face recognition and smart monitoring. If put these models in edge device in actual use, primary work is to keep the accuracy still in a decent level while compressing the model and changing ops to low-bit version. Detailed work can be achieved cooperating with the quantization WG.
+
+#### Size
+
+In ONNX 1.4 release, support for large models (larger than 2GB) and store the data externally is added as a new feature. But in edge device, we do not want such huge models running and consuming the precious memory. There will always be compromises between performance, speed, cost and consumption. There are many ways to compress our models while keeping the accuracy like changing to quantized ops, model compression etc. ONNX Edge WG works to define the compression method for ONNX model and to develope quantized ops.
```

Comment:
We should expand that Size is not only about on-disk/storage size but also about run-time memory utilization. 
Also, the last sentence "ONNX Edge WG works to define the compression method for ONNX model and to develope quantized ops." is outside of scope and it should be deleted.